automation_tools/__init__.py

```diff
@@ -961,6 +961,8 @@ def upstream_install(
             os_version=os_version,
             sam='--sam' if sam else ''
         ))
+    run('yum install -y '
```

Comment:
This works but I was wondering if we should create a PLUGINS constant and list them there, in alphabetical order?

```python
PLUGINS = [
    'ec2',
    'gce', 
    'libvirt', 
    'openstack', 
    'ovirt', 
    'rackspace', 
    'vmware',
]

    run('yum install -y '
           'foreman-{{{0}}}'.format(','.join(PLUGINS))
```